packages/create-skill-app/templates/next/next.config.js

```diff
@@ -5,6 +5,9 @@ const withMDX = require('@next/mdx')({
     providerImportSource: '@mdx-js/react',
   },
 })
+const withTM = require('next-transpile-modules')([
+  '@skillrecordings/skill-lesson',
+])
```

Comment:
updated the templated next config to align with how `total-typescript` is configured